src/actions/movies.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const baseUrl = 'https://reactjs-cdp.herokuapp.com';
+
+export const startMoviesRequest = (query, searchBy) => 
+  (dispatch) => axios.get(`${baseUrl}/movies?search=${query}&searchBy=${searchBy}`)
```

Comment:
While doing, requests to the backend is a common practice in Redux applications to track the states of a request. 
Imagina situation that you're backend is taking long or retrieve an error while fetching your data. It will for sure get lost into your interface.

What we usually do it to track 3 states of a request: FETCHING, SUCCESS and FAILED and set our state in a proper state based on that.

Usually this approach consist in a code more or less like this bellow:

```javascript
export const startMovieRequest = (query, searchBy) => async dispatch => {
    dispatch(requestMovie());

     try {
         const { data } = await axios.get(/* url builder */);

          dispatch(receiveMovie(data));
     } catch (err) {
          dispatch(receiveMovieFailed(err));
     }
}

```